### ISSUE TITLE
question about docu

### ISSUE THREAD

**Issue by realsobek** [bibliocoll/JournalTouch#70]
According to:
`https://github.com/bibliocoll/JournalTouch/wiki/Installation-&-Setup`

after installation I have to do:
`cd data/config`
`cp config-default.php config.php`

Is this still required?
I skipped the steps and it works out, as far as I can see.

Maybe it was the predecessor of:
`data/config/user_config.php`

which gets created after you click on [Save] at
http://journaltouch/admin -- Settings

**Comment by krugar** [comment]
yep, the wiki is still refering to 0.3, that is outdated info. there is some info on
https://github.com/bibliocoll/JournalTouch/blob/master/doc/github/version_0.4_unreleased.md
but we (well, guess that means "i") still need to write proper docs for 0.4 and the migration from 0.3 to 0.4
